fix(headers): Make questions more visible (#78)
uc-cdis/gen3.org: content/resources/faq/_index.md

## Compose Services

<h4>I installed Compose Services, how do I create a program and a project?</h4>

Please refer to corresponding
[Gen3 Documentation](https://gen3.org/resources/operator/#5-programs-and-projects)
to create your first program and project.

> <h4>I created a program, but I see `NotFound` error when I try to create a

Reviewer: "... a `NotFound` error..."

Author: Fixed in ca3cbb7

documentation.

> <h4>User.yaml contains information about program/project, but I still can not

Reviewer: "can not" should be "cannot"

Author: Fixed in ca3cbb7

access it.</h4>

Please ensure you are using `dbgap_accession_number` value as an identifier for
the `auth_id` field, not program's `name` or project's `code`.

<h4>I see a warning about my connection is not being private.</h4>

> 

Reviewer 2: I'd remove "is"

Author: Fixed in 4b5ddf4

certificate instead of generated by our script, this warning will go away.

> <h4>I want to run Compose Services on the server, how to configure

Reviewer 3: ```suggestion
<h4>I want to run Compose Services on the server, how do I configure
```

> <h4>I want to run Compose Services on the server, how do I configure

Reviewer: add a question mark to the end

Author: Fixed in ca3cbb7

Author: Fixed in ca3cbb7

 it.</h4>

To configure Compose Services with your domain, please change the `BASE_URL`
field in the `fence_config.yaml` file, and the `hostname` field in the
`peregrine.json` and `sheepdog.json` files.

> <h4>I'm trying to upload files with gen3-client, but receive an error 

Reviewer 3: ```suggestion
<h4>I'm trying to upload files with the gen3-client, but I receive the error 
```

`x509: certificate signed by unknown authority`.</h4>

> 

Reviewer 2: I'd change "receive" to "received"

Author: Fixed in 4b5ddf4

permissions file.

> <h4>I modified `user.yaml` file. How to update Arborist?</h4>

Reviewer 3: ```suggestion
<h4>I modified the `user.yaml` file. How to update Arborist?</h4>
```

> <h4>I modified the `user.yaml` file. How to update Arborist?</h4>

Reviewer: "How do I update Arborist?"

If you run `gen3 job run usersync` changes should be picked up automatically.
If you don't want to run that command every time you make changes to your
`user.yaml`, you can deploy the cronjob `gen3 job run usersync-cronjob` and
kubernetes will check for changes every hour and update services accordingly.

> <h4>How to change the string "Generic Data Commons" in webpages of my Cloud

Reviewer 3: ```suggestion
<h4>How do I change the title of my webpage from the default "Generic Data Commons"?
```

title at your choice. Then reroll portal with `gen3 kube-setup-portal`.

> <h4>I updated Fence configuration but changes don't appear?</h4>

Reviewer 3: ```suggestion
<h4>I updated the Fence configuration. Why haven't the changes taken effect?</h4>
```

After updating the Fence configuration, run `kubectl delete secret
fence-config`, `gen3 kube-setup-fence` and roll Fence for configuration changes
to take effect with `gen3 roll fence`.

> <h4>GraphiQL on Query page form throwing errors.</h4>

Reviewer 3: ```suggestion
<h4>I receive errors when running queries on the GraphiQL "Query" page, e.g., Cannot query field "field" on type "Root".</h4>
```

> 

Reviewer 2: Is it "GraphiQL" or "GraphQL"?

Reviewer: GraphiQL is the UI, GraphQL is the API :P

Author: In the Windmill it is GraphiQL

You may need to hit the `Switch to Graph Model` button in the upper right
corner.

> <h4>How to run SNS service?</h4>

Reviewer: "How do I run SNS services?"

Author: Fixed in ca3cbb7

Please refer to [Cloud
Automation](https://github.com/uc-cdis/cloud-automation/tree/master/tf_files/aws/modules/data-bucket-queue)
documentation.

<h4>Arborist and Sheepdog don't work as expected, what should I check?</h4>

`kubectl logs <name of pod>` can show you corresponding errors in the logs.

> <h4>I uploaded the new dictionary, what service should I restart?</h4>

Reviewer 3: ```suggestion
<h4>I've updated the data dictionary, what services should I restart for the change to take effect?</h4>
```

First, restart Peregrine `gen3 roll peregrine` and Sheepdog `gen3 roll
sheepdog`.  After these two services restarted, please portal with `gen3 roll
portal`.  Please note, if you are making significant changes to the dictionary,
you will also need to update `portal_app` in the manifest.

<h4>I branched out windmill portal, should I update manifest with a new version of
portal?</h4>

> 

Reviewer 2: I'd recommend putting "the" between "out" and "windmill".  And capitalize "Windmill"

Author: Fixed in 4b5ddf4

> <h4>I branched out the Windmill portal, should I update manifest with a new

Reviewer: I don't 100% understand what "branched out" mean here

Author: Created branch from the repo, forked.

Yes, you can update the manifest to use your version of the portal and use `gen3
kube-setup-portal` or `gen3 roll portal` to changes to take effect.

<h4>How can I update the APP for the portal?</h4>

You can set the `global.portal_app` setting in your manifest to the name of your
windmill configuration file and run `gen3 kube-setup-portal`.

> <h4>What I need to restart if I updated Fence configuration with OAuth

Reviewer: "What do I need to restart..."

Author: Fixed in ca3cbb7

credentials?</h4>

Please run `kubectl delete secret fence-config` and `gen3 kube-setup-fence`.  If
you also updated the `user.yaml` in the folder `apis_configs`, you need to run
`gen3 job run useryaml-job`.

> <h4>I'm storing `user.yaml` in S3 (not locally) and updated it, how to ensure

Reviewer: "how do I ensure"

Author: Fixed in ca3cbb7

## Data Dictionary

> <h4>I'm customizing data schema, where I can find information about the data

Reviewer: "I'm customizing the data schema..."

Author: Fixed in ca3cbb7
